lua/entities/starfall_processor/init.lua

```diff
@@ -73,9 +73,72 @@ function ENT:Think()
 	end
 end
 
+function ENT:GetSendData(toowner)
+	if not self.instance then return end
+
+	local senddata = {
+		owner = self.sfdata.owner,
+		files = {},
+		mainfile = self.sfdata.mainfile,
+		proc = self
+	}
+
+	for k, v in pairs(self.sfdata.files) do senddata.files[k] = v end
+
+	local ppdata = self.instance and self.instance.ppdata
+	if ppdata then
+		if ppdata.serverorclient or (not toowner and ppdata.owneronly) then
+			for filename, code in pairs(senddata.files) do
+				local isserver, isowneronly = ppdata.serverorclient and ppdata.serverorclient[filename] == "server", ppdata.owneronly and ppdata.owneronly[filename]
+				if isserver or (not toowner and isowneronly) then
+					local infodata = {}
+					if isserver then
+						infodata[#infodata + 1] = "-- Server only"
+					end
```

Comment:
With that change to L136 this should be changed to
```lua
if isserver then
    infodata[#infodata + 1] = "--@server"
end
```
So it always gets caught

Reply:
Beautiful